conf/references/mlss_conf.xml

```diff
@@ -11,39 +11,44 @@
       <!-- Metazoa -->
       <genome name="drosophila_melanogaster"/>
       <genome name="caenorhabditis_elegans"/>
-      <genome name="octopus_bimaculoides"/>
       <!-- Fungi -->
       <genome name="saccharomyces_cerevisiae"/>
       <!-- Plants -->
       <genome name="arabidopsis_thaliana"/>
-      <genome name="triticum_aestivum"/>
+      <genome name="oryza_sativa"/>
       <!-- Protists -->
       <genome name="plasmodium_falciparum"/>
     </collection>
 
+    <collection name ="shared_vertebrates">
+      <base_collection name="shared"/>
+      <genome name="mus_musculus"/>
+      <genome name="xenopus_tropicalis"/>
+    </collection>
+
     <collection name="default">
       <base_collection name="shared"/>
       <!-- Vertebrates -->
       <genome name="mus_musculus"/>
-      <genome name="anolis_carolinensis"/>
+      <genome name="thamnophis_elegans"/>
       <genome name="ciona_intestinalis"/>
       <genome name="xenopus_tropicalis"/>
       <genome name="oryzias_latipes"/>
-      <genome name="petromyzon_marinus"/>
+      <genome name="eptatretus_burgeri"/>
       <!-- Metazoas -->
-      <genome name="anopheles_gambiae"/>
-      <genome name="apis_mellifera"/>
-      <genome name="bombyx_mori"/>
-      <genome name="capitella_teleta"/>
-      <genome name="nematostella_vectensis"/>
-      <genome name="adineta_vaga"/>
+      <genome name="daphnia_pulex"/>
+      <genome name="amphimedon_queenslandica"/>
+      <genome name="clytia_hemisphaerica"/>
+      <genome name="lottia_gigantea"/>
+      <genome name="trichoplax_adhaerens"/>
+      <genome name=" mnemiopsis_leidyi"/>
```

Comment:
```suggestion
      <genome name="mnemiopsis_leidyi"/>
```

Reply:
indeed, this is something I completely forgot to do. Thank you for reminding